Core/src/org/sleuthkit/autopsy/logicalimager/dsp/AddLogicalImageTask.java

```diff
@@ -182,70 +274,171 @@ private String addReport(Path reportPath, String reportName) {
         }
         return imagePathToObjIdMap;
     }
-    
+
     @Messages({
         "# {0} - line number", "# {1} - fields length", "# {2} - expected length", "AddLogicalImageTask.notEnoughFields=File does not contain enough fields at line {0}, got {1}, expecting {2}",
         "# {0} - target image path", "AddLogicalImageTask.cannotFindDataSourceObjId=Cannot find obj_id in tsk_image_names for {0}"
     })
-    private void addInterestingFiles(File src, Path resultsPath) throws IOException, TskCoreException {
+    private void addInterestingFiles(File src, Path resultsPath, boolean createVHD) throws IOException, TskCoreException {
```

Comment:
This doesn't appear to be cancellable?

Reply:
I was able to detect the cancel and return, but I don't think there is a rollback for interesting items.

Reply:
No I made a note in the testing story that I don't think we can rollback at this point.